### PR TITLE
Added inactive field display

### PR DESCRIPTION
**Pull request by marcoooo**
Issue #9 fix.

### CODE REVIEW

**Comment by DinerIsmail** [comment]
Looks great! What's the icon though? I would have gone with something like 💤? Also, super tiny but would you mind making the 'I' from 'Inactive' lowercase so it's consistent?

File: hooks/listings/useListings.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useQuery } from 'react-query';
 
 async function fetchListingsRequest() {
-	const response = await fetch('/api/listings');
+	const response = await fetch('/api/listings?');
```
Comment (DinerIsmail) [inline hooks/listings/useListings.jsx:4]: Oh this isn't what I meant with the question mark. I meant where the code was like
`listings.map` to make it into `listings?.map`. The question mark represents optional chaining, and it acts as a null check, basically if the thing before the question mark is null it won't try to run `map` on it and will just return null.

Comment (marcoooo) [reply hooks/listings/useListings.jsx:4]: No problem, I'll correct that then.

**Comment by marcoooo** [comment]
> Looks great! What's the icon though? I would have gone with something like zzz? Also, super tiny but would you mind making the 'I' from 'Inactive' lowercase so it's consistent?

Happy to find another one in the list. I couldn't find a proper list of Names on the web. I used "archived" in the meantime.

**Review by DinerIsmail — APPROVED** [review]
Looks good, it's ready to be merged

**Pull request merged**